examples/nonlocal_games/plot_chsh_game.py

```diff
@@ -0,0 +1,24 @@
+# ruff: noqa: D205, D400, D415
+"""CHSH Game Example
+=================
+
+.. note::
+   This is a sample tutorial to test the layout for the samples gallery project.
+
+This example calculates the classical and quantum values for the CHSH game
+using the toqito library. It demonstrates how to construct the necessary matrices
+and use the XORGame function.
+"""
+
+
+# sphinx_gallery_thumbnail_number = 1
+
+import matplotlib.pyplot as plt
+
+plt.plot([0, 1, 2], [0, 1, 4])  # Example plot
+
+plt.title("CHSH Game")
+
+plt.savefig("thumbnail.png")
```

Comment:
there's not need to save the plot. We will have executable code blocks for these examples.